assets/style.css

```diff
@@ -0,0 +1,781 @@
+    margin-top: 50px;
+    font-size: 100%;
+    box-sizing: border-box;
+    align-items: center;
+  }
+
+  .footer-links-menu li a {
+    margin: 5px;
+    border: 0;
+    vertical-align: baseline;
+    box-sizing: border-box;
+    align-content: center;
+    flex-wrap: nowrap;
+  }
+
+  .footer-legal {
+    width: 100%;
+    height: 100px;
+    padding: 50px 0;
+    background-color: #000;
+    display: block;
+  }
+
+  span img {
+    width: 37px;
+    height: 9px;
+  }
+}
+
+@media only screen and (min-width: 1024px) {
+  /* ====== NAVIGATION ====== */
+  .logo-div,
+  .fa-bars {
+    display: none;
+  }
+
+  .width-1024 {
+    display: block;
+  }
+
+  .width-1023 {
+    display: block;
+  }
+
+  .width-320 {
+    display: none;
+  }
+
+  .first-navbar {
+    background-color: #131313;
+  }
+
+  .split-links,
+  .second-nav-links,
+  .latest-funding-title-links,
+  .footer-links {
+    justify-content: space-between;
+  }
+
+  .nav-flex {
+    width: 100%;
+  }
+
+  .funding-links-right li {
+    padding: 0 5px;
+    border-right: 1px solid #fe4736;
+  }
+
+  .first-navbar ul li {
+    margin-right: 10px;
+  }
+
+  .first-navbar li a {
+    color: #555;
+  }
+
+  .news {
+    color: #fff !important;
+  }
+
+  .first-nav-ul {
+    padding-left: 10px;
+  }
+
+  /* second-nav-bar */
+
+  .second-nav-links-span span {
+    margin-right: 10px;
+  }
+
+  .second-nav-links-span span a {
+    color: rgb(146, 140, 140);
+  }
+
+  .second-nav-links {
+    width: 97%;
+    font-size: 14px;
+    padding: 18px 12px 0;
+    position: relative;
+  }
+
+  .lens-menu {
+    background-color: red;
+    position: absolute;
+    right: 0;
+  }
+
+  .fa-search {
+    position: absolute;
+    right: 10px;
+  }
+
+  /* SHOWCASE */
+  .showcase-section {
+    grid-template-columns: 70% 30%;
+    gap: 1em;
+  }
+
+  .showcase-logo-div {
+    top: -140px;
+  }
+
+  .showcase-logo {
+    display: inline-block;
+    width: 400px;
+  }
+
+  .release-date {
+    float: right;
+    bottom: 150px;
+    right: 20px;
+  }
+
+  .mobile-showcase-content .release-date {
+    display: block;
+  }
+
+  .main-showcase {
+    height: 400px;
+  }
+
+  .aside-showcase {
+    height: 47%;
+  }
+
+  .main-showcase > div h1 {
+    font-size: 40px;
+  }
+
+  .mobile-showcase-content {
+    bottom: 2px;
+    left: 5px;
+  }
+
+  .aside-showcase > div h1 {
+    font-size: 20px;
+  }
+
+  .aside-showcase > div :nth-last-child(1) {
+    display: none;
+  }
+
+  /* LATEST NEWS SECTION */
+  .latest-news-grid,
+  .latest-funding-contents {
+    grid-template-columns: repeat(4, 1fr);
+    gap: 1em;
+  }
+
+  .latest-news-div {
+    flex-direction: column;
+  }
+
+  span img {
+    width: 37px;
+    height: 9px;
+  }
+
+  .latest-news-div img {
+    width: 100%;
+  }
+
+  /* LATEST FUNDING ROUNDS */
+  .width-768 {
+    display: block;
+  }
+
+  .latest-funding-title-links {
+    display: flex;
+  }
+
+  .funding-links-right {
+    display: flex;
+    flex-direction: row;
+  }
```

Comment:
## Using the Same Rule Repetitively[1]
- [  ] You already have a `display: flex;` rule in the line #2 in this file, please use it.
- [ ] Remove all display in this file 
**Example:**

**CSS file**

```CSS
.flex{
  display: flex;
}
```
**Html file**


```HTML
<div class="flex">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_